pkgs/applications/video/multiviewer-for-f1/default.nix

```diff
@@ -90,6 +91,27 @@ stdenvNoCC.mkDerivation rec {
     runHook postInstall
   '';
 
+  passthru.updateScript = writeScript "update-multiviewer-for-f1" ''
+    #!/usr/bin/env nix-shell
+    #!nix-shell -i bash -p curl common-updater-scripts
+    set -eu -o pipefail
+
+    # Get latest API for packages, store so we only make one request
+    latest=$(curl -s "https://api.multiviewer.app/api/v1/releases/latest/")
+
+    # From the downloaded JSON extract the url, version and id
+    link=$(echo $latest | jq -r '.downloads[] | select(.platform=="linux_deb").url')
+    id=$(echo $latest | jq -r '.downloads[] | select(.platform=="linux_deb").id')
+    version=$(echo $latest | jq -r '.version')
+
```

Comment:
I think it would be good to skip the download if the version is already packaged. No need to wastefully download ~150MB.